test/compliance/SELECT.test.js

```diff
@@ -531,6 +531,9 @@ describe('SELECT', () => {
   })
 
   describe('orderby', () => {
+
+    const _localeSort = (a, b) => a === null ? -1 : b === null ? 1 : String.prototype.localeCompare.call(a, b)
```

Comment:
I would expect a check that both `a` and `b` are `null`, but this is a test and only has single `null` row. So ...

Reply:
yes, not needed, but in case someone adds another test I've adapted it 👍